File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        i=0
        j=0
        while(j<len(nums)):
            if(nums[i]==nums[j] and (j-i+1)>2):
                nums.remove(nums[j])
            elif(nums[i]!=nums[j]):
                i +=1
                j +=1
            else:
                j +=1
        
        k = 0
        for i in nums:
            k +=1
        
        return k

File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
        for(int i=2;i<nums.size();i++)
        {
            if(nums[i]==nums[i-1])
            {
                if(nums[i-2]==nums[i-1])
                {
                    nums.erase(nums.begin()+i);
                    i--;
                }
            }
        }
        return nums.size();
    }
};